jadler-jdk/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2012 - 2015 Jadler contributors
+This program is made available under the terms of the MIT License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>net.jadler</groupId>
+        <artifactId>jadler-pom</artifactId>
+        <version>1.1.2-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>jadler-jdk</artifactId>
+    <version>1.1.2-SNAPSHOT</version>
```

Comment:
Remove version to use one from parent. Also provide other elements like scm, name, url, etc. (see pom in other some module).